capa/features/extractors/dnfile/types.py

```diff
@@ -42,15 +42,19 @@ def __repr__(self):
         return str(self)
 
     @staticmethod
-    def format_name(class_: str, namespace: str = "", member: str = ""):
+    def format_name(class_: tuple, namespace: str = "", member: str = ""):
+        if type(class_) == tuple:
+            if len(class_) > 2:
+                class_ = class_[0] + "." + "/".join(class_[1:])
+            else:
+                class_ = f"{class_[0]}.{class_[1]}"
```

Comment:
Please add comments here with examples e.g. like https://github.com/mandiant/capa/blob/94a8b65b480fa0a5944638b0ce3c3dbfd3737bca/capa/features/extractors/dnfile/types.py#L51-L55

Reply:
Added test cases I have seen for the proper string creation.